sllurp/verb/capabilities.py

```diff
@@ -0,0 +1,71 @@
+"""Reset command.
+"""
+
+import logging
+import pprint
+from sllurp.llrp import LLRPReaderConfig, LLRPReaderClient, LLRPReaderState
+from sllurp.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def shutdown(reader, state):
+    host, port = reader.get_peername()
+    logger.info('Shutting down reader %s:%d', host, port)
+    reader.disconnect()
+
+def main(args):
+    if not args.host:
+        logger.info('No readers specified.')
+        return 0
+
+    factory_args = {
+        'start_inventory': False,
+        'reset_on_connect': False,
+    }
+
+    reader_clients = []
+    for host in args.host:
+        if ':' in host:
+            host, port = host.split(':', 1)
+            port = int(port)
+        else:
+            port = args.port
+
+        config = LLRPReaderConfig(factory_args)
```

Comment:
Cool, this uses some of the machinery that @fviard developed. I'll ask them for a review too.

Reply:
Thank you so much for your review!